etcdserver/security/security.go

```diff
@@ -31,8 +35,30 @@ import (
 const (
 	// StorePermsPrefix is the internal prefix of the storage layer dedicated to storing user data.
 	StorePermsPrefix = "/2"
+	RootRoleName     = "root"
+	GuestRoleName    = "guest"
```

Comment:
Guest role took me a second to figure out, might be worth mentioning in a comment that this is the "unauthenticated" role